server/src/test/java/org/apache/druid/server/StatusResourceTest.java

```diff
@@ -85,7 +85,11 @@ private void testHiddenPropertiesWithPropertyFileName(String fileName)
                                                            .map(StringUtils::toLowerCase)
                                                            .collect(Collectors.toSet());
     Set<String> hiddenProperties = new HashSet<>();
-    Splitter.on(",").split(returnedProperties.get("druid.server.hiddenProperties")).forEach(hiddenProperties::add);
+    String hiddenPropertiesString = returnedProperties.get("druid.server.hiddenProperties")
```

Comment:
I'm wondering if we can use jackson to deserialize the `hiddenPropertiesString` to a String array or set.